internal/k8shandler/forwarding.go

```diff
@@ -111,336 +88,17 @@ func (clusterRequest *ClusterLoggingRequest) generateCollectorConfig(extras map[
 	return generatedConfig, err
 }
 
-func (clusterRequest *ClusterLoggingRequest) readClusterName() (string, error) {
```

Comment:
Where did this go?

Reply:
This is moved to `validate_clusterlogforwarderspec.go` at line 368. This was because this method was only ever used when verifying outputs. It was specifically for `CloudWatch`